lib/button.js

```diff
@@ -0,0 +1,326 @@
+/*jslint browser: true, unparam: true, todo: true*/
+/*globals define: true, MutationObserver: false, requestAnimationFrame: false, performance: false, btoa: false, Event: false*/
+'use strict';
+
+export default function (self) {
+  var zIndexTop;
```

Comment:
For future PRs, would you be willing to use `const` or `let` instead? Not a huge deal, because the current code base is still littered with `var`, but we're trying to migrate away, and smaller scoping would make bug-fixing and refactoring in the future easier.

Reply:
ok